### DISCUSSION TITLE
White space below screen in iPad when using external keyboard

### DISCUSSION THREAD

**Discussion by unclecode** [coder/code-server#5446] [General] [answered]
Hi, When I open VScode on iPad, and focus on a text area, a white space appears below the screen due to the keyboard. This happens when using an external keyboard.

**Comment by unclecode** [comment, marked as answer]
This is already resolved. on ipad, Settings > General > Keyboard and then disable shortcuts.

**Comment by lprhodes** [comment]
@unclecode  this does solve the issue but it then prevents two finger scroll on the trackpad

**Reply by unclecode** [reply]
I hvnt face with that issue. I can scroll using two fingers

**Reply by simonfalke-01** [reply]
I have the same issue

**Reply by jsjoeio** [reply]
@lprhodes @Brandonli-13 can you reproduce with github.dev? If so, you can raise an issue upstream
